client/components/swipeable/style.scss

```diff
@@ -0,0 +1,32 @@
+@import '@wordpress/base-styles/mixins';
+.swipeable__container {
+	width: 100%;
+	height: 100%;
+	margin-left: auto;
+	margin-right: auto;
```

Comment:
It is usually a good practice for a component to stop at its `border-box`, and let the component's consumer decided the occupied space and especially the margin behaviour.

I would suggest moving the `margin-*` rules (and potentially the `height` too) to where this component is consumed (i.e. the Dot Pager component)

Reply:
Good idea. I removed the margin-* properties.